### DISCUSSION TITLE
Manage graphic (chart, pie...) width and height

### DISCUSSION THREAD

**Discussion by Zorg77127** [sqlpage/SQLPage#270] [Q&A]
Hello Community,

Juste using SqlPage for a few days... Looks very promising, and very practical for what I want.
Maybe I missed something , but is it possible to manage width and height for a generated graphic (chart or pie , etc...) ?

I'm' building a chart like this : 

```sql
SELECT 'hero' AS component, "blabla" AS title;
select 
      'chart'    as component,
    "blabla" as title,
    'area'     as type,
    'indigo'  as color,
    4            as marker,
'ToolBar'    as toolbar,
0               as logarithmic,
'Date'        as xtitle,
'number'   as ytitle,
    TRUE     as time;
select datetime(timezone) as x,
       CAST(datas as integer) as y
from XXX;
```

I want to increase or decrease the size "byd default" of the generated chart. Is it possible ?

Thanks a lot in advance for your help.

**Comment by lovasoa** [comment]
hello! 

It's currently not possible to change the size of a graph. It shouldn't be difficult to implement a custom height parameter, though. Do you want to try and open a pull request? You could take inspiration from the map component which already has this. 

If you need multiple small graphs next to each other, you can also use the card component with the embed parameter. See the last example on https://sql.ophir.dev/documentation.sql?component=card#component

**Comment by lovasoa** [comment]
Hi ! Coming back at this a short while later, I realize that the `height` property is actually already implemented; it was simply missing from the documentation.

```sql
select 
    'chart'               as component,
    500                   as height;
select x, y from my_data;

**Reply by Zorg77127** [reply]
oh great ! works perfectly for me, thanks for this info .
